src/Runtime/OMResize.inc

```diff
@@ -0,0 +1,353 @@
+#ifdef __cplusplus
+#include <cassert>
+#else
+#include <assert.h>
+#endif
+
+#include <math.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "onnx-mlir/Runtime/OMTensor.h"
+
+/**
+ * The runtime implementation of ONNXResizeOp is manually translated python
+ * implementation: onnx/onnx/backend/test/case/node/resize.py (onnx v1.9.0)
+ * This implementation can be improved in efficiency.
+ **/
+
+static void linear_coeffs(float ratio, float *coeffs_buffer, int mode) {
+  coeffs_buffer[0] = 1 - ratio;
+  coeffs_buffer[1] = ratio;
+}
+
+static void nearest_coeffs(float ratio, float *coeffs_buffer, int mode) {
+  /* integer ratio is handled outside */
+  switch (mode) {
+  case 0: // round_prefer_float
+    coeffs_buffer[0] = ratio <= 0.5;
```

Comment:
`coeffs_buffer` is an array of float, but the values stored into it are actually of type `_Bool`. Suffest to change its data type to `_Bool` for C or `bool` for C++.

Reply:
The original code takes the conversion of bool to float. I change the code to make it explicit: `coeffs_buffer[0] = ration <= 0.5 ? 1 : 0;`, for all the assignments.